zingolib/src/lib.rs

```diff
@@ -46,31 +37,29 @@ pub async fn load_clientconfig_async(
     // Create a Light Client Config
     let config = ZingoConfig {
         server_uri: Arc::new(RwLock::new(server)),
-        chain: match info.chain_name.as_str() {
+        chain: match chaintype {
```

Comment:
Now that we're not parsing the chain type from a string, we can use our own strongly-typed `zingoconfig::ChainType` enum instead of a string here.